pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -220,6 +230,9 @@ def _plan_pinned_builds(self):
         if not self.build_plan.build_images:
             run_details.update_description('Images: not building.<br/>')
 
+        elif self.mass_rebuild:
+            run_details.update_description('Mass image rebuild (more than half) - invoking serializing semaphore')
+
         elif self.build_plan.images_included:
             run_details.update_description(f'Images: building {self.build_plan.images_included}.<br/>')
```

Comment:
we have over 200 images in total even not a mass_rebuild with 100 is still a lot, I suggest only shows len(self.build_plan.images_included)